target/linux/ipq806x/base-files/etc/hotplug.d/firmware/11-ath10k-caldata

```diff
@@ -73,6 +77,10 @@ case "$FIRMWARE" in
 		caldata_extract "0:ART" 0x5000 0x2f20
 		ath10k_patch_mac $(mtd_get_mac_ascii 0:APPSBLENV ethaddr)
 		;;
+	askey,rt4230w)
+		caldata_extract "ART" 0x5000 0x7f1f
+		ath10k_patch_mac $(macaddr_add $(mtd_get_mac_binary ART 0x6) +2)
```

Comment:
No address found in ART 0x5006?

Reply:
The MAC addresses match up with stock fw. Confirmed with ifconfig and WiFi Analyzer on android.